pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -542,6 +542,7 @@ func (k *Bootstrapper) restartCluster(cfg config.ClusterConfig) error {
 	cmds := []string{
 		fmt.Sprintf("%s phase certs all --config %s", baseCmd, conf),
 		fmt.Sprintf("%s phase kubeconfig all --config %s", baseCmd, conf),
+		fmt.Sprintf("sudo systemctl restart kubelet"),
```

Comment:
I think this will break CloudShell.

Reply:
We still run systemd within kic even in Cloud Shell -- I tried the following with this binary in a boosted Cloud Shell VM and it worked as expected:

```shell
minikube start
minikube stop
minikube addons enable dashboard
minikube start # Dashboard pods came up in about 20 seconds
```

Do you think we still need this change? I don't totally understand it's purpose right now.

Reply:
You are totally right here. Sorry for the misunderstanding.

Anyways, restarting kubelet here is not entirely useful, as the new kubelet configuration hasn't been written to disk. Allegedly, the kubelet phase will write the updated config to disk, and restart it, but if you find that kubeadm does not actually restart kubelet for whatever reason, feel free to keep this statement.

Reply:
That makes sense, I've removed it!

Reply:
Are you going to add the kubeadm kubelet phase in it's place?